drivers/storage/portworx/component/pvccontroller.go

```diff
@@ -309,17 +309,14 @@ func (c *pvcController) createDeployment(
 		"--leader-elect=true",
 		"--controllers=persistentvolume-binder,persistentvolume-expander",
 		"--use-service-account-credentials=true",
+		"--leader-elect-resource-name=portworx-pvc-controller",
+		fmt.Sprintf("--leader-elect-resource-namespace=%s", cluster.Namespace),
```

Comment:
I haven't followed all the threads on slack about this. But have we gone back and verified that older k8s versions also have these arguments? This would fail to come up in those versions if the args are absent.

Reply:
we have bump the min supported k8s version to 1.22

Reply:
i have checked all k8s versions 1.21+ have following two args. so let's just use the following args to use different resource name and not specify what type of leader election - leave to default. 

         command = append(command, "--leader-elect-resource-name=portworx-pvc-controller")
	command = append(command, fmt.Sprintf("--leader-elect-resource-namespace=%s", cluster.Namespace))